partitioned-table.md

```diff
@@ -1356,78 +1352,21 @@ mysql> explain select * from t1 where id < 150;
 
 From the above query results, you can see that the `Union` operator in the execution plan disappears while the partition pruning still takes effect and the execution plan only accesses `p0` and `p1`.
 
-`dynamic` mode makes execution plans simpler and clearer. Omitting the Union operation can improve the execution efficiency and avoid the problem of Union concurrent execution. In addition, `dynamic` mode also solves two problems that cannot be solved in `static` mode:
```

Comment:
Could we add some known limitations, for example, 

1) 'Prepared plan cache' cannot be applied with dynamic pruning mode.
2) Pruning with 'like ...' condition is not supported yet.

Reply:
@bb7133 could you take a new look? I added the pruning limitation under the "Partition pruning" section and "prepared statement plan cache" under the Dynamic prune mode.